### ISSUE TITLE
When running the latest version of Grouparoo, the app won't start with error "file is a reserved param"

### ISSUE THREAD

**Issue by evantahler** [grouparoo/grouparoo#487]
**Describe the bug**
When running the latest version of Grouparoo, the app won't start with error "file is a reserved param".  This is an Actionhero error.  This error should have been fixed by https://github.com/grouparoo/grouparoo/pull/469

**To Reproduce**
1. Follow the instructions in the README - make a `package.json` + `.env` and run `npm install`
2. when you run `npm start` the error occcurs

**Expected behavior**
The App Starts with no error

**Screenshots**
<img width="746" alt="Screen Shot 2020-07-20 at 5 05 06 PM" src="https://user-images.githubusercontent.com/303226/88079926-3c14fb00-cb33-11ea-9f41-61782eb8232b.png">

**Stack Trace**
```
Using ts-node version 8.10.2, typescript version 3.9.7
modified your runtime environment with /Users/evan/workspace/grouparoo/tmp/.env
2020-07-21T16:28:29.408Z - notice: Starting Grouparoo 0.1.5 on node.js v12.16.3
2020-07-21T16:28:34.044Z - alert: Failed to load /Users/evan/workspace/grouparoo/tmp/node_modules/@grouparoo/core/api/src/actions/files.ts stacktrace=Error: input `file` in action `file:create` is a reserved param
```

**Environment (please complete the following information):**
 - OSX

CC @annaleeherrera

**Comment by evantahler** [comment]
It's interesting that this error doesn't occur in the `example-app`

**Comment by evantahler** [comment]
The reason that `app-example` continues to work is that the project was generated with actionhero 23.0.4, and `@grouaroo/core` requires `actionhero@^23.0.4`.... which allows the new `actionhero@^23.0.5`.  

Form https://docs.npmjs.com/cli/update:
> As of npm@2.6.1, the npm update will only inspect top-level packages. Prior versions of npm would also recursively inspect all dependencies. To get the old behavior, use npm --depth 9999 update.

Solution: Release a new version of `@grouparoo/core` with the fixes linked above
To prevent this from happening again: have `app-example` run periodically and do a fresh-install so we notified if second-level dependencies of Grouparoo break something.

**Comment by evantahler** [comment]
Partially solved by https://github.com/grouparoo/app-example/pull/28

**Comment by evantahler** [comment]
Closed by the [0.1.6](https://github.com/grouparoo/grouparoo/releases/tag/v0.1.6) release.  

Thanks again @annaleeherrera!
